boltstub/bolt_protocol.py

```diff
@@ -227,7 +227,7 @@ class Bolt1Protocol(BoltProtocol):
     server_agent = "Neo4j/3.3.0"
 
     def get_auto_response(self, request: TranslatedStructure):
-        if request.name == "HELLO":
+        if request.name in "INIT":
```

Comment:
```suggestion
        if request.name == "INIT":
```
using `in` will make a theoretical request with name `IN` or `INI` pass this check, which I don't think is something we want.

Reply:
Whoopsiedoodle. I must have stumbled over my keyboard :smile: